pkg/ds/farm.go

```diff
@@ -327,13 +336,16 @@ func (dsf *Farm) handleDSChanges(changes dsstore.WatchedDaemonSets, quitCh <-cha
 
 			if _, ok := err.(consulutil.AlreadyLockedError); ok {
 				if _, ok := dsf.children[dsFields.ID]; !ok {
-					dsLogger.Debugln("Lock on daemon set was already acquired by another farm")
+					dsf.logger.Infof("Lock on daemon set '%v' was already acquired by another farm", dsFields.ID)
 					continue
 				}
+				dsf.logger.Info("Lock on daemon set '%v' was already acquired by this farm", dsFields.ID)
```

Comment:
Updated, the code now follows this pattern, it will only acquire the lock if the child does not exist it its map.